microsoft-edge/webview2/how-to/debug-devtools.md

```diff
@@ -18,6 +18,8 @@ When you're using a WebView2 app, there are several ways to open DevTools:
 *  Press `Ctrl`+`Shift`+`I`.
 *  Right-click the page and then select `Inspect`.
 
+Apps can also use the `CoreWebView2.OpenDevToolsWindow` / `ICoreWebView2::OpenDevToolswindow` API to programmatically open a DevTools window if they remove those hotkeys and the context menu items. In the event none of those options are possible, adding `--auto-open-devtools-for-tabs` to additional browser arguments via an environment variable or registry key will open a DevTools window when a WebView2 is created.
```

Comment:
I don't know the right way to add links to these pages. Can someone share the right syntax for linking to:
https://docs.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2.opendevtoolswindow and https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2#opendevtoolswindow

Reply:
the tagging is complex and specialized including applying your good link-text design.  i'll tag these in VS Code and push a commit